### DISCUSSION TITLE
記事を書きたい

### DISCUSSION THREAD

**Discussion by okuramasafumi** [rubima/magazine.rubyist.net#390] [記事を書きたい]
とりあえず宣言だけしておきます（宣言しておかないと書かないと思ったので）。

内容としては、RubyConfに登壇したりするので、そのあたりの経験とかを書けたらいいなと思ったりしています。

**Comment by kakutani** [comment]
いかがでしょう…???

**Comment by neko314** [comment]
何度か、都度具体的にこの記事を書きたいと申し出てくださったりして執筆いただいてるので、こちらは閉じますね。また書いていただける時にはこれまでと同じように進められたらと思います🙂
